uitools/toolkitwidgets/toolkitwidgets.pri

```diff
@@ -13,22 +13,26 @@
 # limitations under the License.
 include($$PWD/../common/common.pri)
 
-QT += widgets webenginewidgets svg
+android || ios {
+    message("toolkitwidgets.pri is not usable on mobile platforms.")
+} else {
+    QT += widgets webenginewidgets svg
```

Comment:
I should've mentioned this before, but I don't think we need the else block. It can all remain unindented. This is the qmake version of https://clang.llvm.org/extra/clang-tidy/checks/readability/else-after-return.html haha.

Reply:
There is not return in qmake (I guess). Without `else`, that means only the message is displayed and there is a build error for mobile platforms due to webenginewidgets.

Reply:
Oh my mistake, you are right. I thought `message` was `error`.  🤦‍♂️ 

Use `error` and that will halt the build on mobile platforms. Would that work?

Reply:
Done